var/spack/repos/builtin/packages/meshtool/package.py

```diff
@@ -16,6 +16,7 @@ class Meshtool(MakefilePackage):
 
     version("master", branch="master", preferred=True)
     # Version to use with openCARP releases
+    version("oc14.0", commit="867431d")
```

Comment:
Can we use the full commit hash? Why do most versions have the same commit hash?